README.md

```diff
@@ -2,3 +2,12 @@
 SDl Overview Guide content for the [SDL Developer Portal](https://smartdevicelink.com/en/guides/sdl-overview-guides/introduction/).
 
 This is where the documentation for describing most of SDL features will be held.
+
+## Updating the RPC Spec and Protocol Spec
+
+To update the RPC Spec and Protocol Spec documentation to the latest version, please run the following commands:
+
+```bash
+cd doc_retriever
```

Comment:
Specify doc retriever is in the overview guides repo

Reply:
Resolved in https://github.com/smartdevicelink/sdl_overview_guides/pull/54/commits/4e1ea02bb303ea172caf788ddb13c9b076c57718